src/mscorlib/src/System/Threading/Tasks/TPLETWProvider.cs

```diff
@@ -24,7 +24,7 @@ namespace System.Threading.Tasks
     [EventSource(
         Name = "System.Threading.Tasks.TplEventSource",
         Guid = "2e5dba47-a3d2-4d16-8ee0-6671ffdcd7b5",
-        LocalizationResources = System.CoreLib.Name)]
+        LocalizationResources = "FxResources.System.Private.CoreLib.SR")]
```

Comment:
It caused System.Diagnostics.Tracing tests to fail on my machine. That's why I have fixed it.